default/python/AI/ProductionAI.py

```diff
@@ -1772,6 +1435,8 @@ def _try_enqueue(
     opinion = building_type.get_opinions()
     locations = []
     preferred_locations = []
+    if isinstance(candidates, int):  # isinstance(candidates, ObjectId) does not work, at least not in python-3.9
```

Comment:
`ObjectId` is just typing wrapper around int for better readability, most API does not return it. 

PlanetId not ObjectId

Reply:
Yes, of course it should be PlanetId.
As far as I understood, NewType will actually create a class in python 3.10, so I guess it would be possible to specify it in is_instance as well, wouldn't it?